admin_guide/managing_projects.adoc

```diff
@@ -94,3 +102,60 @@ or:
 To request a new project, fill out the project request form located at
 https://internal.example.com/openshift-project-request.
 ----
+
+[[using-node-selectors]]
+== Using Node Selectors
+
+Node selectors are used in conjunction with labels to control pod placement. 
```

Comment:
Alright I can see that potential confusion. However "with labels" is ambiguous as to where the labels go (nodes? pods? something else?). How about: "Node selectors are used in conjunction with labels on nodes ... " (or, labeled nodes)?